docs/advanced/verifying-binaries.md

```diff
@@ -286,14 +288,14 @@ There are many ways to generate a SHA-256 hash, but here are a few:
 
     * In a terminal:
 
-            $ certutil -hashfile <binary file> SHA256
+            $ certutil -hashfile <your binary file> SHA256
 
 
 * macOS or Linux
 
     * In a terminal:
 
-            $ sha256sum <binary file>
+            $ sha256sum <your binary file>
```

Comment:
this command doesn't exist on a (default) macos install.

The command to use there is `shasum -a 256`.

Reply:
Thanks! Fixed.